### ISSUE TITLE
AppImage not creating desktop files

### ISSUE THREAD

**Issue by xgdgsc** [marktext/marktext#1291]
Ubuntu16.04下0.15的AppImage怎么不提示新建desktop了, 0.14的旧的desktop在gnome shell里搜索不到，启动marktext也不能favorite到dock ， 删了desktop文件0.15也不提示新建
 
### Description
- [ ] Can you reproduce the issue? 

### Steps to reproduce

1.  run appimage of 0.15
 

**Expected behavior:**

Prompt to integrate with desktop and .desktop file is created in user home .local

**Actual behavior:**

Nothing happens with desktop integration and marktext is launched, not able to pin on gnome shell dock.
 
 
### Versions

- Mark Text: 0.15
- Operating system: Ubuntu 16.04.6 LTS with gnome shell

**Comment by DutchPete** [comment]
@xgdgsc : right-click on the AppImage icon in its folder, in the context menu click on *Send to*, then choose *Desktop (create link)*. There will be a shortcut on the desktop. This how it works on Linux Mint XFCE, I imagine there is a similar possibility on Ubuntu. Isn't that enough?

**Comment by xgdgsc** [comment]
Thanks. I' m using another machine with ubuntu 18.04 and KDE now with same issue. Dolphin doesn' t have this send to option. From my own electron experience it might be related to some packager of electron.
